app/src/main/java/com/seafile/seadroid2/data/DatabaseHelper.java

```diff
@@ -59,6 +61,13 @@
     private static final String DIRENTS_CACHE_COLUMN_PATH = "path";
     private static final String DIRENTS_CACHE_COLUMN_DIR_ID = "dir_id";
 
+    public static final String ENCKEY_TABLE_NAME = "EncKey";
+
```

Comment:
It might be better to store the secret in the Android Authenticator (where the account tokens are already stored). That would make them a bit harder to access for attackers.